media/css/firefox/firstrun/firstrun_quantum.less

```diff
@@ -0,0 +1,542 @@
+}
+
+:root {
+  --animation-curve: linear;
+  // --animation-curve: cubic-bezier(.73,.08,.94,.33);
+  // --backwards: cubic-bezier(.49,-0.41,.96,.67);
+}
+
+body {
+    margin: 0;
+    overflow: hidden;
+}
+
+* {
+    font-family: 'Fira Sans Light';
+}
+
+#scene {
+    height: 100vh;
+    width: 100vw;
+}
+
+.background {
+    transform: scale(1.5);
+    position: absolute;
+    right: 0;
+    left: 0;
+    margin: auto;
+}
+
+.white-overlay {
+    fill: #FFFFFF;
+    height: 100vh;
+    width: 100vw;
+    position: absolute;
+    opacity: 0;
+}
+
+.st1 {
+    fill: url('#SVGID_1');
+    height: 100vh;
+    width: 100vw;
+}
+
+.st2 {
+    fill: url('#SVGID_2');
+    height: 70vh;
+    width: 85vw;
+    bottom: 0;
+}
+
+.st3 {
+    opacity: 0.5;
+    fill: url('#SVGID_2');
+    height: 48vh;
+    width: 75vw;
+    bottom: 0;
+}
+
+.st4 {
+    opacity: 0.6;
+    fill: #005FE7;
+    height: 26vh;
+    width: 100vw;
+    bottom: 0;
+}
+
+.st5 {
+    opacity: 0.6;
+    fill: #002079;
+    height: 51vh;
+    width: 100vw;
+}
+
+.st6 {
+    opacity: 0.8;
+    fill: url('#SVGID_1');
+    height: 73vh;
+    width: 100vw;
+}
+
+.st7 {
+    opacity: 0.3;
+    fill: url('#SVGID_1');
+    margin: 0;
+    height: 59vh;
+    width: 60vw;
+}
+
+.st8 {
+    opacity: 0.6;
+    fill: #002079;
+    height: 51vh;
+    width: 100vw;
+}
+
+.cls-2 {
+    height: 100vh;
+    width: 100vw;
+    position: absolute;
+    fill:url('#linear-gradient');
+    mix-blend-mode:color;
+    opacity: 0;
+}
+
+#firefox-logo {
+    background: url('/media/img/firefox/firstrun-quantum/quantum-logo.svg');
+    width: 80px;
+    height: 80px;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    opacity: 0;
+    position: relative;
+    float: left;
+    clear: both;
+}
+
+#fxa {
+    border: none;
+    position: relative;
+    width: 420px;
+}
+
+#fxa-iframe-config {
+    background: #fff;
+    display: inline-block;
+    margin-left: 77px;
+    transform: scale(.8);
+    opacity: 0;
+    padding-bottom: 20px;
+}
+
+.fxaccounts-container {
+    bottom: 0;
+    color: #fff;
+    height: 485px;
+    left: 0;
+    margin: auto;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 980px;
+    z-index: 10;
+}
+
+#left-divider {
+    position: relative;
+    float: left;
+    clear: both;
+    width: 483px;
+    margin-top: 80px;
+    margin-top: 50px;
+
+    #title,
+    .content,
+    a {
+        opacity: 0;
+        transform: translateY(-5px);
+    }
+
+    .content {
+        font-size: 19px;
+        line-height: 150%;
+        max-width: 352px;
+        margin-bottom: 60px;
+    }
+
+    a {
+        color: #fff;
+        display: block;
+        font-size: 19px;
+        font-weight: 400;
+    }
+
+    #title {
+        font-size: 42px;
+        margin: 0 0 10px;
+    }
+}
+
+#skip-button {
+    -moz-appearance: none;
+    background-color: #0996f8;
+    border-radius: 3px;
+    border: 1px solid #0670cc;
+    box-shadow: 0 0 0 0 transparent;
```

Comment:
I think `box-shadow: none;` also removes a shadow, but this may not be needed since I'm not sure this button has a shadow to remove.